pkgs/development/libraries/kde-frameworks/extra-cmake-modules/setup-hook.sh

```diff
@@ -59,19 +59,15 @@ xdgDataSubdirs=( \
     "wallpapers" "applications" "desktop-directories" "mime" "appdata" "dbus-1" \
 )
 
-ecmHostPathSeen=( )
+declare -gA ecmHostPath
```

Comment:
nit: how about `ecmSeenHostPaths`
I would like to add an `s` to the variable since it's an array.
I also liked the fact that `Seen` was included in the name to mark as the path as seen.
It's entirely a matter of style, so if you feel strongly around it, feel free to ignore.